src/misc/portal.ts

```diff
@@ -59,6 +59,21 @@ export class Portal extends Component<any, any> {
       }
     });
   }
+  /**
+   * Override to revert back to a classic Component's structure
+   *
+   * @override
+   */
+  __callWillUnmount() {
+    if (this.el && this.portal && this.portal.elm) {
+      this.el.appendChild(this.portal.elm);
+      const vnode = this.__owl__.vnode;
+      vnode!.children = [this.portal];
+      this.portal = null;
+      this.doTargetLookUp = true;
+    }
+    super.__callWillUnmount();
```

Comment:
this should be done at the beginning. otherwise subcomponents will not be at the correct place when they are unmounted